murdock/tests/test_api.py

```diff
@@ -383,7 +383,7 @@ def test_restart_job_not_allowed(restart):
 
 
 @pytest.mark.usefixtures("admin_allowed")
-@pytest.mark.parametrize("result,code", [([], 404), ([test_job_finished], 200)])
+@pytest.mark.parametrize("result,code", [([], 200), ([test_job_finished], 200)])
```

Comment:
The delete jobs endpoint now always returns with code 200 so there's no need to a specific variable. The parametrize could be simplified:
```suggestion
@pytest.mark.parametrize("result", [[], [test_job_finished]])
```